lib/known_bugs.pm

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2015-2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use base Exporter;
+use Exporter;
+use strict;
+use testapi;
+use main_common;
+
+our @EXPORT = qw(
+  create_list_of_serial_failures
+);
+
+sub create_list_of_serial_failures {
+    my $serial_failures = [];
+
+    # To add a known bug simply copy and adapt the following line:
+    # push @$serial_failures, {type => soft/hard, message => 'Errormsg', pattern => quotemeta 'ErrorPattern' }
+
+    # Detect bsc#1093797 on aarch64
+    if (is_sle('=12-SP4') && check_var('ARCH', 'aarch64')) {
```

Comment:
@DrMullings We still need the version utils module

Reply:
I just added them ^^